esrally/mechanic/mechanic.py

```diff
@@ -266,7 +266,7 @@ def receiveMsg_StartEngine(self, msg, sender):
         self.car, _ = load_team(self.cfg, msg.external)
 
         # In our startup procedure we first create all mechanics. Only if this succeeds we'll continue.
-        hosts = self.cfg.opts("client", "hosts")
+        hosts = self.cfg.opts("client", "hosts")()
```

Comment:
I think it would be better to use an explicit property name here instead of `()`?